Graph/1466ReorderRoutestoMakeAllPathsLeadtotheCityZero.py

```
@@ -0,0 +1,39 @@
+          for value in self.undirected_graph[node]:
+              if value in self.visited:
+                 continue
+              if node not in self.directed_graph[value]:
+                 self.count+=1
+              self.helper(value)
+          return  
+          
+          
+          
+            
+class TestApp:
```

Flake8 (7.2.0)

[error] too many blank lines (4)

(E303)